File: JAVA QUESTION/StudentScore.java
import java.util.Scanner;
public class StudentScore {
	public static void main(String[] args) {
	Scanner input = new Scanner(System.in);

	System.out.print("How many students do you have?: ");
	int studentNumber = input.nextInt();

	System.out.print("How many subject do they offer?: ");
	int subjectNumber = input.nextInt();

	
	for (int count = 1; count <= studentNumber; count++) {
		System.out.println("\nWhat is the name of student " + count + "?");
		System.out.print("Enter student name: ");
		String name = input.next();

		double studentTotal = 0;


		for (int counter = 1; counter <= subjectNumber; counter++) {
			System.out.print("\nWhat is the name of subject " + counter + ":");
			String subjectName = input.next();

			System.out.print("Enter score for " + subjectName + "(1-100)");
			double score = input.nextDouble();
			studentTotal = studentTotal + score;

}
		double average = studentTotal / subjectNumber;

	System.out.println("Name: " + name);
	System.out.println("Total Score: " + studentTotal);
	System.out.println("Average: " + average);
	System.out.println("============================");


}

}
}

File: PYTHON QUESTION/student_score.py
student_number = int(input("How many students? "))
subject_number = int(input("How many subjects? "))

for count in range(1, student_number + 1):
    print(f"\nWhat is the name of student {count}?")
    name = input("Enter student name: ")

    student_total = 0

    for counter in range(1, subject_number + 1):
        subject_name = input(f"Enter name of subject {counter}: ")
        score = float(input(f"Enter score for {subject_name} (0-100): "))
        student_total = student_total + score

    average = student_total / subject_number

    print("Name:", name)
    print("Total Score:", student_total)
    print("Average:", average)
    print("===========================")